Jenkinsfile

```diff
@@ -74,26 +74,26 @@ try {
           disable_aslr = 'setarch x86_64 -R'
 
           // Run cmake once in isolation and build jemalloc to avoid race conditions with autoconf (#1413)
-          sh "mkdir clang-debug && cd clang-debug &&                                                       ${cmake} ${ccache} ${debug}   ${clang}          .. && make -j libjemalloc-build"
```

Comment:
Why only for debug? I don’t get what we are doing here from the comment.

Reply:
better?

Reply:
So it’s not about debug. It’s about running autoconf “anyhow” and afterwards everything works?

Reply:
correct

Reply:
Remove the empty line + replace “really” with “using debug here”)?

For my understanding: for cmake you need a new cmake run every time you change from debug to release. For automake, that’s not the case?

Reply:
Correct. I don't want to mention debug here because it has nothing to do with debug.

If you don't insist, I#d rather merge this once it's green. This is *the* PR needed to unfuck CI.

Reply:
The empty line still bothers me though. :D